### PR TITLE
Feature: cc-0041 update readme for backend

### PR DESCRIPTION
**Pull request by Jzeva**
Update the Readme file for back end which woud be pretty similar to our front end.

### CODE REVIEW

**Review by tatoMa — COMMENTED** [review]
Do you think we need to add the docker composer command here?

**Comment by Jzeva** [comment]
> Do you think we need to add the docker composer command here?

After merging your pull request, the docker composer command has been updated in the readme file. I can make it more specific.
image

File: README.md [hunk 1]
```diff
@@ -35,6 +40,113 @@ $ npm run test:e2e
...
+- Pull Request Title Example: "Feature/cc 0042 a user can log in     sign up for my account"
+- Tutorial About How To Create A Pull Request: [Link](www.google.com)
+      </td>   
+   </tr>
+      <tr>
+      <td width="500"><b>4. Specific Code Convention</b></td>
+   </tr>
+   <tr>
+      <td>
+- Basically we will use double double quotes `""`instead of single quotes`''` in the front end project.
```
Comment (GitHubEason) [inline README.md:144]: Do you mind to delete a double pls

Comment (Jzeva) [reply README.md:144]: Didn't we decide use double quotes in back-end?

**Review by Fuchih — APPROVED** [review]

**Review by EmilyGong111 — APPROVED** [review]

**Pull request merged**